tests/core/engine_adapter/docker-compose.yaml

```diff
@@ -50,47 +50,54 @@ services:
     volumes:
       - ./trino/catalog:/etc/trino/catalog
 
-  trino_datalake_metastore_db:
+  trino_metastore_db:
```

Comment:
I refactored this slightly to make it take up less memory. Rather than having an entire Postgres instance per metastore, I changed it to a single Postgres instance with a database per metastore